Sources/App/Controllers/ExploreController.swift

```diff
@@ -25,6 +25,9 @@ final class ExploreController {
         // Get all categories and find out if one of them is selected.
         let categories = try Category.all().select(selected: queryCategory)
 
+        let accessoryCount = categories.reduce(0, {$0 + $1.accessoriesCount})
```

Comment:
I think we could improve the efficiency of this count. Since at line 42 we query anyway all the accessories, we can directly `count` that array instead of doing a query for each category. Do you agree?

Reply:
Yeah I see what you mean here.  I think I overlooked the part where using the `accessoriesCount` attribute was going to do a query each time 😄 

This doesn't help the case where someone selects a category but we could do the same thing here that you mentioned in your other comment  e.g.  put all accessories into a variable, count them, then filter down to the category that was picked.

Do you think that's better than doing the count of each category?

Reply:
Yes, that’s exactly what I meant.